app/models/server/models/Rooms.js

```diff
@@ -283,6 +284,28 @@ export class Rooms extends Base {
 		return this.find(query, { fields: { ts: 1, departmentId: 1, open: 1, servedBy: 1, metrics: 1, msgs: 1 } });
 	}
 
+	async getMostRecentAverageChatDurationTime(numberMostRecentChats, department) {
```

Comment:
please use the specific raw model for raw methods: https://github.com/RocketChat/Rocket.Chat/blob/develop/app/models/server/raw/Rooms.js